api.yml

```diff
@@ -17,7 +17,7 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credentials:
+  /credentials/issue:
```

Comment:
```suggestion
  /credentials:
```

Reply:
-1 to injecting verbs into the URL. There was discussion about not doing this in separate issues previously.